ansible_collections/arista/avd/roles/eos_designs/doc/network-services-v4.0.md

```diff
@@ -463,6 +470,13 @@ svi_profiles:
         # Activate or deactivate IGMP snooping | Optional, default is true
         igmp_snooping_enabled: < true | false >
 
+        # Structured configuration and eos cli commands rendered on router_bgp.vlans
+        # This configuration will not be applied to vlan aware bundles
+        bgp:
+          raw_eos_cli: |
+            < multiline eos cli >
+          structured_config: < dictionary >
+
```

Comment:
cosmetic: our documentation has two l2vlans configured and we put some of the options in 1 and some of the options in the other which makes it hard to have a full view of the keys (same for `igmp_snooping_enabled` btw). I am wondering if this would not be better to have everything under one.

Reply:
Hmmm .. actually this is only in the v4.0 doc so ignore this